param_persist/sqlalchemy/models/__init__.py

```diff
@@ -0,0 +1,15 @@
+"""The __init__.py file for the models package in sqlalchemy.
+
+This file was generated on July 30, 2020
+"""
+__author__ = "Gage Larsen"
+__copyright__ = "Copyright: (c) Aquaveo 2020"
+__maintainer__ = "Gage Larsen"
+__email__ = "glarsen@aquaveo.com"
+
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+from param_persist.sqlalchemy.models.instance_model import InstanceModel  # NOQA: F401, E402
```

Comment:
Does importing these Classes create a circular dependency?

Reply:
Not that i could find.

Reply:
There was until i moved the second two imports below the base